src/parsers/pluscal.ts

```diff
@@ -23,6 +23,7 @@ export class TranspilerStdoutParser extends ProcessOutputHandler<DCollection> {
     private readonly filePath: string;
     private errMessage: string | null = null;
     private nextLineIsError = false;
+    private nowParsingAddedLabels = false; //should probably be a state machine thing
```

Comment:
Is the comment a TODO? If so, could you provide more detail and make it clearer?